test/0.8.9/self-owned-steth-burner.test.js

```diff
@@ -0,0 +1,698 @@
+  let oracle, lido, burner
+  let treasuryAddr
+  let dao, acl, operators
+  let compositeBeaconReceiver
+
+  beforeEach('deploy lido with dao', async () => {
+    const lidoBase = await LidoMock.new({ from: deployer })
+    oracle = await LidoOracleMock.new({ from: deployer })
+    const depositContract = await DepositContractMock.new({ from: deployer })
+    const nodeOperatorsRegistryBase = await NodeOperatorsRegistry.new({ from: deployer })
+
+    ;({ dao, acl } = await newDao(appManager))
+
+    // Instantiate a proxy for the app, using the base contract as its logic implementation.
+    let proxyAddress = await newApp(dao, 'lido', lidoBase.address, appManager)
+    lido = await LidoMock.at(proxyAddress)
+
+    // NodeOperatorsRegistry
+    proxyAddress = await newApp(dao, 'node-operators-registry', nodeOperatorsRegistryBase.address, appManager)
+    operators = await NodeOperatorsRegistry.at(proxyAddress)
+    await operators.initialize(lido.address)
+
+    // Set up the app's permissions.
+    await acl.createPermission(voting, lido.address, await lido.PAUSE_ROLE(), appManager, { from: appManager })
+    await acl.createPermission(voting, lido.address, await lido.MANAGE_FEE(), appManager, { from: appManager })
+    await acl.createPermission(voting, lido.address, await lido.MANAGE_WITHDRAWAL_KEY(), appManager, { from: appManager })
+    await acl.createPermission(voting, lido.address, await lido.BURN_ROLE(), appManager, { from: appManager })
+    await acl.createPermission(voting, lido.address, await lido.SET_TREASURY(), appManager, { from: appManager })
+    await acl.createPermission(voting, lido.address, await lido.SET_ORACLE(), appManager, { from: appManager })
+    await acl.createPermission(voting, lido.address, await lido.SET_INSURANCE_FUND(), appManager, { from: appManager })
+
+    await acl.createPermission(voting, operators.address, await operators.MANAGE_SIGNING_KEYS(), appManager, { from: appManager })
+    await acl.createPermission(voting, operators.address, await operators.ADD_NODE_OPERATOR_ROLE(), appManager, { from: appManager })
+    await acl.createPermission(voting, operators.address, await operators.SET_NODE_OPERATOR_ACTIVE_ROLE(), appManager, { from: appManager })
+    await acl.createPermission(voting, operators.address, await operators.SET_NODE_OPERATOR_NAME_ROLE(), appManager, { from: appManager })
+    await acl.createPermission(voting, operators.address, await operators.SET_NODE_OPERATOR_ADDRESS_ROLE(), appManager, {
+      from: appManager
+    })
+    await acl.createPermission(voting, operators.address, await operators.SET_NODE_OPERATOR_LIMIT_ROLE(), appManager, { from: appManager })
+    await acl.createPermission(voting, operators.address, await operators.REPORT_STOPPED_VALIDATORS_ROLE(), appManager, {
+      from: appManager
+    })
+    await acl.createPermission(depositor, lido.address, await lido.DEPOSIT_ROLE(), appManager, { from: appManager })
```

Comment:
Do we need all this permissions for this test? Can we reduce lido boilerplate to focus only on `SelfOwnedStETHBurner` test case?

Reply:
For sure. Thx!